witnet/prices/o/OpUsdtPrice.js

```diff
@@ -76,6 +76,7 @@ const request = new Witnet.Request()
   .addSource(digifinex)
   .addSource(gateio)
   .addSource(kucoin)  
+  .addSource(okex)
```

Comment:
Does this line belong in this PR?

Reply:
i guess okex source was instantiated but not added to the script on a previous pr

Reply:
indeed, okex source was not included on the op/usdt pf deployed on testnet:
- https://witnet.network/search/897fff3fb6b5d5f7efde75d9795e2c808cb9b17606f6fa30d20057a1b7b94532

but, it has been included when deployed on mainnet:
- https://witnet.network/search/0d2aa9389aff208af93170148170010ae2489edb1f776009961d3d1113b2cdf0

Reply:
therefore, op/usdt-6 needs to be upgraded on optimism testnet